tensorflow/workspace.bzl

```diff
@@ -249,6 +283,12 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
       actual = "@gmock_archive//:gtest_main",
   )
 
+  native.git_repository(
```

Comment:
This is also needed by the ops research tools, that depend on the gflags library.